### PR TITLE
Kill final pieces of http api from helm chart

### PR DESCRIPTION
**Pull request by chainlink**

### CODE REVIEW

File: etc/helm/pachyderm/values.schema.json [hunk 1]
```diff
@@ -566,9 +563,6 @@
                             "type": "string"
                         }
                     }
-                },
-                "usesRoot": {
```
Comment (jrockway) [inline etc/helm/pachyderm/values.schema.json:570]: Oh, what was this?

**Review by jrockway — APPROVED** [review]
I wondered what the HTTP API port was.  Now I know -- a blackhole ;)

File: etc/helm/pachyderm/values.schema.json (hunk 1 above)
Comment (chainlink) [reply etc/helm/pachyderm/values.schema.json:570]: that was killed in another PR, but looks like the schema for it got missed, I can break that out into another cleanup PR

Comment (jrockway) [reply etc/helm/pachyderm/values.schema.json:570]: Nah, doing it here is fine.  api port won't be coming back ;)

**Review by jrockway — APPROVED** [review]

**Pull request merged**